Automattic/woocommerce-payments
# Remove payouts rename notice overlay.

Fixes https://github.com/Automattic/woocommerce-payments/issues/9704

#### Changes proposed in this Pull Request









* Remove this overlay which was added in this PR: https://github.com/Automattic/woocommerce-payments/pull/9680

<img width="504" alt="Screenshot 2024-12-25 at 05 03 17" src="https://github.com/user-attachments/assets/8d85bd20-f18e-46c2-891d-85e21b5b08d0" />



#### Testing instructions





* Open either payments overview page, deposits list page, or deposit details page.
* Confirm that you will *not* see the payout rename spotlight notice even though option `wcpay_payouts_rename_notice_dismissed` doesn't exist (run `delete_option( 'wcpay_payouts_rename_notice_dismissed' )` on [WP Console](https://wordpress.org/plugins/wp-console/) to delete that option).
* On base branch, the notice overlay will show if option `wcpay_payouts_rename_notice_dismissed` does not exists.

-------------------


**Post merge**
